src/main/java/nextstep/fp/StreamStudy.java

```diff
@@ -28,6 +29,13 @@ public static void printLongestWordTop100() throws IOException {
         List<String> words = Arrays.asList(contents.split("[\\P{L}]+"));
 
         // TODO 이 부분에 구현한다.
+        words.stream()
+            .filter(word -> word.length() > 12)
+            .sorted(StreamStudy::compare)
+            .distinct()
+            .map(word -> word.toLowerCase())
+            .limit(100)
+            .forEach(System.out::println);
```

Comment:
```suggestion
        String joinedWords = words.stream()
            .filter(word -> word.length() > 12)
            .sorted(StreamStudy::compare)
            .distinct()
            .map(word -> word.toLowerCase())
            .limit(100)
            .collect(Collectors.joining("\n"));
        System.out.println(joinedWords);
```
표준 출력을 사용할 일이 거의 없겠지만
System.out.println 같은 I/O 작업을 100건 호출하는 방식보다는
문자열을 만들고 한번에 출력하는 것이 더 좋을 것 같아요